src/templates/default_nativeapp/README.md

```diff
@@ -0,0 +1,32 @@
+## Introduction
+
+This is the default project template for a Snowflake Native Apps project. It contains minimal code meant to help you set up your first application instance in your account quickly.
+
+### Project Structure
+| File Name | Purpose |
+| --------- | ------- |
+| ./README.md | The current file you are looking at, meant to guide you through a native apps project. |
+| app/setup_script.sql | Contains SQL statements that are run when an account installs or upgrades an application. |
```

Comment:
I assume this is initial work. Do you plan to bootstrap python, java or scala project? Having a real "project structure" would be awesome.

Reply:
There are two snowflake-provided templates for our MVP:
1. This PR aims to provide a barebones template, with no extension code, i.e. python/java/scala code. This shows all the code residing in the setup_script.sql. The reason is to provide a lean native app project that is one-click ready to be installed in the snowflake account. 
2. There will be an additional project template that contains a structure that shows the extension code files. This will be provided as part of a separate ticket under our epic.